src/scrapper/spiders/course_unit_group.py

```diff
@@ -0,0 +1,121 @@
+import scrapy
+import hashlib
+from ..items import CourseGroup, CUCG
+from ..database.Database import Database
+
+
+class CourseUnitSpider(scrapy.Spider):
```

Comment:
`course_unit_group` spider class should not have the same name as the `course_unit_spider.py` spider